src/timeMachine/components/Vis.tsx

```diff
@@ -90,6 +93,26 @@ const TimeMachineVis: FC<Props> = ({
       giraffeResult.table.getColumnType('_value') !== 'number' &&
       !!giraffeResult.table.length)
 
+  const dispatch = useDispatch()
```

Comment:
i'd say move this down into the 

```
if (isFlagEnabled('graphColorMapping')) {
```

block since it's only used there. I think this will make it slightly easier to follow since it's used where it's needed, and it will move a variable initialization out of code that doesn't need it.

Reply:
As per our slack discussion, this would cause the linter to throw an error because dispatch needs to be called from the top of the component.